src/dbg/x64dbg.cpp

```diff
@@ -675,13 +677,140 @@ static DWORD WINAPI loadDbThread(LPVOID hEvent)
     return 0;
 }
 
-static WString escape(WString cmdline)
+static String escape(String cmdline)
 {
-    StringUtils::ReplaceAll(cmdline, L"\\", L"\\\\");
-    StringUtils::ReplaceAll(cmdline, L"\"", L"\\\"");
+    StringUtils::ReplaceAll(cmdline, "\\", "\\\\");
+    StringUtils::ReplaceAll(cmdline, "\"", "\\\"");
     return cmdline;
 }
 
+class CommandlineArguments : public ArgumentParser
+{
+public:
+    String filename;
+    std::vector<std::string> extraCmdline;
+    String currentDir;
+    String pid;
+    String tid;
+    String event;
+    String command;
+    String commandFile;
+    bool help = false;
+
+    CommandlineArguments() : ArgumentParser("x64dbg")
+    {
+        addPositional("filename", filename, "Filename of program to debug");
+        addPositional("currentdir", currentDir, "Current working directory of new process");
+        addExtra(extraCmdline);
+
+        addString("-p", pid, "Process ID to attach to.");
+        addString("-a", pid, "Alias for -p.");
+        addString("-tid", tid, "Thread Identifier (TID) of the thread to resume after attaching.");
+        addString("-e", event, "Handle to an Event Object to signal on attach");
+
+        // TODO: Allow repeating multiple -c arguments

```

Comment:
You can separate commands with a `;` to execute multiple commands in a row (but as you probably saw they are not executed like the script and do not wait for the debugger to be paused again)